Assets/HoloToolkit-Tests/SpatialMapping/Scripts/PlaneFindingTest.cs

```diff
@@ -111,7 +111,7 @@ private void OnDrawGizmosSelected()
                     UnityEditor.Handles.DrawLine(corners[1], corners[2]);
                     UnityEditor.Handles.DrawLine(corners[1], corners[3]);
                     UnityEditor.Handles.DrawLine(corners[2], corners[3]);
-                    UnityEditor.Handles.ArrowCap(0, center, Quaternion.FromToRotation(Vector3.forward, normal), 0.4f);
+                    UnityEditor.Handles.ArrowHandleCap(0, center, Quaternion.FromToRotation(Vector3.forward, normal), 0.4f, EventType.ignore);
```

Comment:
This was ignored because the arrows are for conveying information and are not supposed to be modifiable.